walmart_open.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec", "~> 3.0.0.beta2"
-  spec.add_development_dependency "debugger", "~> 1.6"
+  spec.add_development_dependency "byebug", "~> 8.2.5"
```

Comment:
Can you remove this change? I'd rather discuss this separately in a different issue/PR.